### PR TITLE
MELS

### PR DESCRIPTION
**Pull request by manuroe**
The summary is not yet computed

### CODE REVIEW

File: MatrixKit/Models/Room/MXKRoomBubbleCellData.m [hunk 1]
```diff
@@ -664,6 +694,14 @@ - (MXKEventFormatter *)eventFormatter
     return nil;
 }
 
+#pragma mark - Bubble collapsing
+
+- (BOOL)collaspseWith:(id<MXKRoomBubbleCellDataStoring>)cellData
```
Comment (giomfo) [inline MatrixKit/Models/Room/MXKRoomBubbleCellData.m]: typo collapseWith

File: MatrixKit/Models/Room/MXKRoomBubbleCellDataWithAppendingMode.m [hunk 2]
```diff
@@ -260,7 +260,7 @@ - (NSAttributedString*)attributedTextMessage
 {
     @synchronized(bubbleComponents)
     {
-        if (!attributedTextMessage.length && bubbleComponents.count)
+        if (self.hasAttributedTextMessage && !attributedTextMessage.length && bubbleComponents.count)
```
Comment (giomfo) [inline MatrixKit/Models/Room/MXKRoomBubbleCellDataWithAppendingMode.m]: Can we remove the check on bubbleComponents.count here?

File: MatrixKit/Models/Room/MXKRoomDataSource.m [hunk 3]
```diff
@@ -2196,6 +2311,63 @@ - (void)processQueuedEvents:(void (^)(NSUInteger addedHistoryCellNb, NSUInteger
...
+                    }
+                    while ((nextBubbleData = nextBubbleData.nextCollapsableCellData));
+
+                    // Build the summary string for the serie
+                    bubbleData.collapsedAttributedTextMessage = [self.eventFormatter attributedStringFromEvents:events withRoomState:bubbleData.collapseState error:nil];
+
+                    // Release collapseState objects, even the one of collapsableSerieAtStart.
+                    // We do not need to keep its state because if an collapsable event comes before collapsableSerieAtStart,
+                    // we will take the room state of this event.
+                    if (bubbleData != collapsableSerieAtEnd &&
```
Comment (giomfo) [inline MatrixKit/Models/Room/MXKRoomDataSource.m]: "bubbleData != collapsableSerieAtEnd" is always true when "(bubbleData == collapsableSerieAtStart && collapsableSerieAtStart != collapsableSerieAtEnd)" is satisfied.

**Comment by ara4n** [comment]
should probably s/Serie/Series/g for language-consistency :)

**Comment by manuroe** [comment]
oh. Series is singular in French. Will do s/Serie/Series/g

**Pull request merged**